docusaurus.config.js

```diff
@@ -0,0 +1,75 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Truss Engineering Playbook',
+  tagline: '',
+  url: 'https://trussworks.github.io',
+  baseUrl: '/Engineering-Playbook/',
+  organizationName: 'trussworks',
+  trailingSlash: false,
+  onBrokenLinks: 'warn',
+  onBrokenMarkdownLinks: 'warn',
```

Comment:
With this one and the line above, should we have this `error` rather than `warn`? There shouldn't be too many dead links, and we could fix these. 

On second thought, maybe it makes more sense to turn this on in a separate PR, but I still stand by that it should `error` rather than `warn` in the future.

Reply:
I have it on a list to fix in a future PR. Unfortunately, there _are_ a handful of dead links - not because Docusaurus broke them, but because they had died long ago 😩 . I definitely agree that it should be an error instead of a warning though